docs/install-openshift.md

```diff
@@ -55,16 +55,33 @@ The following is the simplest recommended configuration for the `DatadogAgent` i
         appSecret:
           keyName: app-key
           secretName: datadog-secret
+      clusterName: <CLUSTER_NAME>
       criSocketPath: /var/run/crio/crio.sock
       kubelet:
         # This is needed if the kubelet certificate is self-signed.
         # Alternatively, the CA certificate used to sign the kubelet certificate can be mounted.
         tlsVerify: false
+    features:
+      apm:
+        enabled: true
+        hostPortConfig:
+          enabled: true
+        # Unix Domain Socket uses a `hostPath` volume, prevented by OpenShift default SecurityContextConstraints for applicative pods.
+        # More information: https://docs.datadoghq.com/containers/troubleshooting/admission-controller/?tab=datadogoperator#openshift
+        unixDomainSocketConfig:
+          enabled: false
+      dogstatsd:
+        hostPortConfig:
+          enabled: true
+        unixDomainSocketConfig:
+          enabled: false
     override:
       nodeAgent:
         # In OpenShift 4.0+, set the hostNetwork parameter to allow access to your cloud provider metadata API endpoint.
         # This is necessary to retrieve host tags and aliases collected by Datadog cloud provider integrations. 
         hostNetwork: true
+        image:
+          name: gcr.io/datadoghq/agent:latest
```

Comment:
do you mind removing the image override to `latest` (for dca as well)? (we should also remove this from the public docs.) using `latest` isn't recommended as we do some semver checks in the code. thanks!

Reply:
Adressed in [`3f2f9e2` (#1370)](https://github.com/DataDog/datadog-operator/pull/1370/commits/3f2f9e223eba944ab7359d4c8293b795f3a0eaa4)